skycfg.go

```diff
@@ -215,12 +219,21 @@ func loadImpl(ctx context.Context, opts *loadOptions, filename string) (starlark
 		cache[modulePath] = nil
 		globals, err := starlark.ExecFile(thread, modulePath, moduleSource, opts.globals)
 		cache[modulePath] = &cacheEntry{globals, err}
+
+		for name, val := range globals {
+			if strings.HasPrefix(name, "test_") && val.Type() == "function" {
```

Comment:
I think we'll want to use a Go type cast here, so we don't need to make assumptions about Starlark type names.

```go
if !strings.HasPrefix(name, "test_") {
    continue
}
if fn, ok := val.(starlark.Callable); ok {
    tests = append(tests, &Test{
        callable: fn,
    })
}

```

Reply:
Good call.